csi.proto

```diff
@@ -228,19 +231,22 @@ message VolumeCapability {
 message CapacityRange {
   // Volume must be at least this big. This field is OPTIONAL.
   // A value of 0 is equal to an unspecified field value.
-  uint64 required_bytes = 1;
+  // The value of this field MUST NOT be negative. 
+  int64 required_bytes = 1;
```

Comment:
Do we want to get rid of the use of underscores while we're at it?  Granted that's mostly my own pedantic annoyance :)  and it would be a separate patch but just asking.

Reply:
Hi @j-griffith,

Please leave the underscores. The tell language generators how to camel case things.